pkg/artifact_builder/artifact_builder.go

```diff
@@ -150,15 +150,26 @@ func (ab *ArtifactBuilder) RetagImages(
 			}
 		}
 
-		repoUrl := strings.Split(registry.GetRepoUrl(), "/")[1]
+		parts := strings.Split(registry.GetRepoUrl(), "/")
```

Comment:
worth adding a test for this?

Reply:
There's one already, but it didn't detect the issue.